README.md

```diff
@@ -275,20 +275,27 @@ ShopifyApp.configure do |config|
 end
 ```
 
-If you'd rather implement your own controller then you'll want to use the WebhookVerfication module to verify your webhooks:
+If you'd rather implement your own controller then you'll want to use the WebhookVerfication module to verify your webhooks, example:
 
 ```ruby
 class CustomWebhooksController < ApplicationController
   include ShopifyApp::WebhookVerification
 
   def carts_update
-    SomeJob.perform_later(shopify_domain: shop_domain, webhook: params)
-    head :ok
+    params.try(:permit!)
```

Comment:
can you explain what this is for?

Reply:
Ah, it was just to match closely to how the `WebhookController` is currently doing it in the lib. However, we're not filtering anything so permit wouldn't make sense, I could remove it if that would be best?

Reply:
Oh nvm I see now - it permits everything. I think the `try` threw me off. I bet that was to try and support old versions of Rails which we don't do anymore. Can you update this in both places to be `params.permit!`

Reply:
Sure, this is done. That makes sense, thanks.